src/lib/ngsi/AttributeList.cpp

```diff
@@ -41,18 +41,19 @@
 */
 std::string AttributeList::render(Format format, const std::string& indent, bool comma)
```

Comment:
This potentially could conflict with render changes in the PR https://github.com/telefonicaid/fiware-orion/pull/891, but I think they should easy to fix.

NTC